### ISSUE TITLE
Issue with UCR Ham dataset

### ISSUE THREAD

**Issue by jandrulus** [tejaslodaya/timeseries-clustering-vae#7]
Hi tejaslodaya!
First of all, congratulations for the repository, I think it is great!
I have an issue withe UCR "Ham" dataset. When I run the code, with the following hyperparameters: 

hidden_size = 90
hidden_layer_depth = 1
latent_length = 20
batch_size = 32
learning_rate = 0.0005
n_epochs = 20
dropout_rate = 0.5
optimizer = 'Adam' # options: ADAM, SGD
cuda = True # options: True, False
print_every= 30
clip = True # options: True, False
max_grad_norm = 5
loss = 'MSELoss' # options: SmoothL1Loss, MSELoss
block = 'LSTM' # options: LSTM, GRU

I get the error attached. 

![error](https://user-images.githubusercontent.com/33958578/73658233-47af5b00-4694-11ea-8bf7-03d712ce452f.png)

It seems that test_loader object is empty. Please, can you take a look? I have no problems with other UCR datasets. Thank you in advance, kind regards!
